### ISSUE TITLE
iOS apps using defold-luasec are rejected by Apple

### ISSUE THREAD

**Issue by britzl** [sonountaleban/defold-luasec#4]
When submitting an app that uses defold-luasec to Apple it gets rejected with the following reason:

"The app references non-public symbols in MyCoolProject: _getcontext, _makecontext, _setcontext"

According to [this issue](https://github.com/openssl/openssl/issues/2545) it's because "setcontext is used in async_posix.h and getcontext, makecontext is used in async_posix.c.
async_posix.h/c are both in the openssl-1.1.0c"

The solution is to rebuild OpenSSL using the `no-async` flag. [This project](https://github.com/x2on/OpenSSL-for-iPhone) seems to simplify the build process for OpenSSL for iOS and TVOS platforms.

**Comment by dossy** [comment]
Is this as simple as getting a new `no-async` build of OpenSSL for iOS and replacing the files in `defold-luasec/luasec/lib/ios/lib{crypto,ssl}.a`?

**Comment by dossy** [comment]
Yup, I rebuilt OpenSSL 1.1.0e with `no-async` and rebuilt my Defold project, and submitted to the App Store and it was accepted.

**Comment by britzl** [comment]
Good that you got it sorted. I have a fork where I've done the same thing: https://github.com/britzl/defold-luasec
